### PR TITLE
[BUGFIX] Fix intercom icon not showing up

### PR DESCRIPTION
**Pull request by seawatts**
Remove deferReadinessUntilLoad config
Add test for rendering icon

~~@mike-north how do you feel about having an acceptance test actually go out and get the external file? I'm not a huge fan of doing this but it does help ensure that things are actually being rendered.~~ 

Had some weird issues with getting this to work with previous ember versions so for now I just removed the acceptance test.

### CODE REVIEW

**Comment by mbbertino** [comment]
@seawatts FYI, this fixes #37

File: client/intercom-shim.js [hunk 1]
```diff
@@ -1,9 +1,19 @@
 (function() {
-  /* globals define, Intercom */
```
Comment (mike-north) [inline client/intercom-shim.js]: Why is this needed? You seem to be pulling the global off of the `window` object consistently

File: client/intercom-shim.js [hunk 2]
```diff
@@ -12,32 +22,24 @@
...
-      i.q.push(args);
-    };
-    w.Intercom = i;
-  }
-
   generateModule('intercom', {
-    default: w.Intercom,
+    default: function() {
+      return window.Intercom.apply(null, arguments);
+    },
```
Comment (mike-north) [inline client/intercom-shim.js]: This shouldn't be necessary. Can you explain what's going on here?

File: client/intercom-shim.js (hunk 1 above)
Comment (seawatts) [inline client/intercom-shim.js]: I think my editor might have removed that. Do you think we should use `window.Intercom` or just `Intercom`? I don't really have a strong preference either way.

File: client/intercom-shim.js (hunk 2 above)
Comment (seawatts) [inline client/intercom-shim.js]: I was trying a lot of different things to get the icon to show up and after debugging through this it looked like the babel transpiler was doing something weird and it would loose it's context. From there inside the Intercom call it did not have any of the previous properties set like the `appId`

Comment (mike-north) [inline client/intercom-shim.js]: I'd like to reproduce this weirdness. Any tips or instructions?

File: client/intercom-shim.js (hunk 1 above)
Comment (mike-north) [inline client/intercom-shim.js]: Prefer not to use `window` due to 

* It causing work for us in the future in order to take advantage of tree-shaking (unused dependency removal)
* `window` doesn't exist when rendering on the server-side with FastBoot, and attempting to access it will cause the rendering to fail (crash)

Comment (seawatts) [inline client/intercom-shim.js]: Great points! I'll change it =)

File: client/intercom-shim.js (hunk 2 above)
Comment (seawatts) [inline client/intercom-shim.js]: There is some weird behavior when you run `ember serve` when running the test dummy app. If you load the main page, then click the link. There are no errors but the icon doesn't actually show up. However, if you refresh the page it will show up. With this PR this fixes this weird behavior
